src/test/ui/did_you_mean/issue-93210-ignore-doc-hidden.rs

```diff
@@ -0,0 +1,24 @@
+#[derive(Default)]
+pub struct A {
+    #[doc(hidden)]
+    pub hello: i32,
+    pub bye: i32,
+}
+
+#[derive(Default)]
+pub struct B {
+    pub hello: i32,
+    pub bye: i32,
+}
+
+fn main() {
+    A::default().hey;
+    //~^ ERROR no field `hey` on type `A`
+    //~| NOTE unknown field
```

Comment:
I'm not sure why this note is needed, it doesn't show up in the stderr output :thinking: